src/test_typing_extensions.py

```diff
@@ -5163,7 +5157,7 @@ class C:
         A.x = 5
         self.assertEqual(C.x, 5)
 
-    @skipIf(sys.version_info[:2] in ((3, 9), (3, 10)), "Waiting for bpo-46491 bugfix.")
+    @skipIf(sys.version_info[:2] == (3, 10), "Waiting for bpo-46491 bugfix.")
```

Comment:
While we're here, let's maybe update this to a GitHub reference? It looks like the old BPO issue was migrated to https://github.com/python/cpython/issues/90649, so that would be

```suggestion
    @skipIf(sys.version_info[:2] == (3, 10), "Waiting for CPython gh-90649 bugfix.")
```